MAINTAINERS.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+The oneDPL project provides algorithms that are implemented using parallel patterns and bricks implemented
+for the various backends supported to allow oneDPL to run on CPU, GPU, and FPGA platforms. The API areas
+which a developer is a lead maintainer are listed below.
```

Comment:
Should we have a sentence like the following?

"Questions, PRs, etc. which do not fit into these categories can tag all maintainers, or look to the git blame information to choose the best subset of this list.

Reply:
Done.